pkg/cmd/experimental/tokens/whoami.go

```diff
@@ -29,14 +37,13 @@ func NewCmdWhoAmI(clientCfg *clientcmd.Config) *cobra.Command {
 	return cmd
 }
 
-func whoami(token string, clientCfg *clientcmd.Config) {
-	osCfg := clientCfg.OpenShiftConfig()
+func whoami(token string, clientCfg *kclient.Config, cmd *cobra.Command) {
 	// This will be pulled out of the auth config file after https://github.com/GoogleCloudPlatform/kubernetes/pull/2437
```

Comment:
Go ahead and do this cleanup while you're here.  The token from the bound clientConfig should be used.